code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -176,6 +176,8 @@
 	var/warding_aura = 0
 	var/recovery_aura = 0
 
+	var/regen_power = 0 //Increases by 0.1 every time you regen. Resets when you take damage.
```

Comment:
Since it can be annoying to find the actual value (-5) can you reverse these? So it starts at -5 and then resets to 0 on damage?

Reply:
It needs to be -5 as that controls the cooldown of it. If you want, I can add a define.

Reply:
I think you misunderstood, im happy with -5 for timing im just suggesting that you invert the logic so that finding or tuning the value would always happen here instead of in the `take_damage` proc.